.circleci/config.yml

```diff
@@ -196,6 +177,68 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
+  cas1_e2e_tests:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.34.1-focal
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    steps:
+      - run:
+          name: Clone E2E repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-approved-premises-e2e.git .
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - node/install-packages
+      - run:
+          name: E2E Check
+          command: |
+            npm run test
+      - store_artifacts:
+          path: playwright-report
+          destination: playwright-report
+      - store_artifacts:
+          path: test-results
+          destination: test-results
+      - slack/notify:
+          event: fail
+          channel: << pipeline.parameters.alerts-slack-channel >>
+          template: basic_fail_1
+  cas3_e2e_tests:
+    parallelism: 4
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    docker:
+      - image: cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+    parameters:
+      environment:
+        type: string
+        default: dev
+    steps:
+      - run:
+          name: Checkout e2e repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui.git .
+      - run:
+          name: install
+          command: apt-get install xz-utils
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - node/install-packages
```

Comment:
Is this our script or something that we get out of the cypress/browsers:node16.14.2-slim-chrome100-ff99-edge image, what packages does it install that's not covered by npm install? Excuse the silly question.

Reply:
It's a CircleCI orb thing I think, it handles caching and the like too, as well as running `npm install`